pkg/pillar/cmd/zedrouter/server.go

```diff
@@ -140,28 +145,47 @@ func getDoneChan(bridgeName string, bridgeIP string) (chan<- struct{}, <-chan st
 	return val.doneChan, val.ackChan, exists
 }
 
+func getTCP80(match string) string {
+	// XXX debug collector
+	cmd := "netstat -antwp | grep " + match
```

Comment:
ough! is this really the best way to do this -- screenscraping netstat output? isn't there a nice go package to track this type of stuff @eriknordmark ?

Reply:
This is just to get some debug output, so if ithe screen scraping breaks it will have no functional impact. 
FWIW I haven't found any golang package to extract the TCP connections from netstat/the kernel.